akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala

```diff
@@ -66,6 +69,11 @@ final class ClusterSettings(val config: Config, val systemName: String) {
     }
   }
 
+  /**
+   * @deprecated Specific to [[akka.cluster.AutoDown]] should not be used anywhere else, instead
+   *   ``Cluster.downingProvider.downRemovalMargin`` should be used as it allows the downing provider to decide removal
+   *   margins
+   */
```

Comment:
yes, that would be good, and we also include the version number when it was introduced, see other places